lib/any.js

```diff
@@ -75,7 +88,7 @@ function configure(query, state) {
   var index = -1
 
   while (++index < pseudos.length) {
-    if (pseudo.needsIndex.indexOf(pseudos[index].name) > -1) {
+    if (needsIndex.indexOf(pseudos[index].name) > -1) {
```

Comment:
```suggestion
    if (pseudo.needsIndex.indexOf(pseudos[index].name) > -1) {
```

this change can be reverted